.github/workflows/tag_release.yml

```diff
@@ -26,9 +26,9 @@ jobs:
       env:
         CHANGELOG_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     - name: Run GoReleaser
-      uses: goreleaser/goreleaser-action@108bc4d
+      uses: goreleaser/goreleaser-action@v2
       with:
-        version: 'v0.135.0'
+        version: latest
         args: release --rm-dist --release-notes=RELEASE_CHANGELOG.md
       env:
         GOVERSION: ${{ env.GOVERSION }}
```

Comment:
@phamann @kailan @doramatadora I'm not sure if any of you know if the below secret needs to be updated from `RELEASE_GITHUB_TOKEN` to just `GITHUB_TOKEN`?

I read through https://docs.github.com/en/actions/reference/authentication-in-a-workflow but it doesn't mention a `RELEASE_GITHUB_TOKEN` which makes me wonder if that is a legacy reference from back when we were using the older version of goreleaser.

Reply:
Looking at the terraform provider which uses v2 of goreleaser and that's using `GITHUB_TOKEN` https://github.com/fastly/terraform-provider-fastly/blob/master/.github/workflows/release.yml#L40